include/openssl/bio.h.in

```diff
@@ -174,9 +174,9 @@ extern "C" {
 
 # ifndef OPENSSL_NO_KTLS
 #  define BIO_get_ktls_send(b)         \
-     BIO_ctrl(b, BIO_CTRL_GET_KTLS_SEND, 0, NULL)
+     (BIO_ctrl(b, BIO_CTRL_GET_KTLS_SEND, 0, NULL) > 0)
 #  define BIO_get_ktls_recv(b)         \
-     BIO_ctrl(b, BIO_CTRL_GET_KTLS_RECV, 0, NULL)
+     (BIO_ctrl(b, BIO_CTRL_GET_KTLS_RECV, 0, NULL) > 0)
```

Comment:
This is actually slightly problematic. This changes the meaning of a public macro. Applications compiled against old headers may behave incorrectly even if they are linked against an up-to-date and fully patched library.

I'm not sure how we should handle that.

Reply:
IMO we can add documentation of that. The change is backwards compatible itself. I.e., if the app handles -1 return it won't be broken by this change.